src/Categories/Category/Instance/SimplicialSet/Properties.agda

```diff
@@ -0,0 +1,212 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Level
+open import Function using (_$_)
+
+module Categories.Category.Instance.SimplicialSet.Properties (o ℓ : Level) where
+
+open import Data.Empty.Polymorphic
+open import Data.Nat using (ℕ)
+open import Data.Fin using (Fin; _<_; fromℕ)
+open import Data.Fin.Patterns
+open import Data.Product using (proj₁)
+
+import Relation.Binary.PropositionalEquality as Eq
+
+open import Categories.Category
+open import Categories.Category.Instance.SimplicialSet
+open import Categories.Category.Instance.Simplex
+
+open import Categories.Functor
+open import Categories.Functor.Construction.Constant
+open import Categories.Functor.Construction.LiftSetoids
+
+open import Categories.NaturalTransformation
+
+open import Categories.Yoneda
+
+private
+  module Y = Functor (Yoneda.embed Δ)
+
+-- Some useful notation for a simplicial set
+ΔSet : Set (suc o ⊔ suc ℓ)
+ΔSet = Category.Obj (SimplicialSet o ℓ )
+
+-- The standard n-simplex.
+Δ[_] : ℕ → ΔSet
+Δ[_] n = LiftSetoids o ℓ ∘F Y.F₀ n
+
+--------------------------------------------------------------------------------
+-- Boundaries of the Standard Simplicies
+--
+-- The basic idea here is that we will build up boundary of 'Δ[ n ]' by considering
+-- all of the morphisms 'Δ[ m , n ]' that factor through some face map 'face i : Δ[ n - 1 , n ]'
+
+-- The indexing here is a bit tricky, but this is the price we pay to avoid 'pred'
+-- A 'Boundary m n' represents some set of maps into 'Δ[ ℕ.suc n ]' that factor through
+-- a face map. To make this indexing more obvious, we use the suggestively named variable 'n-1'.
+record Boundary (m n-1 : ℕ) : Set where
+  field
```

Comment:
maybe use
```agda
private
  n : ℕ
  n = ℕ.suc n-1
```
so as to be less confusing?

Reply:
Good idea! Done